### PR TITLE
Feature/include default libs#272

### PR DESCRIPTION
**Pull request by alekseytsekov**

### CODE REVIEW

File: doc/developer-documentation/aeproject-cli/compile.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ You can specify compiler using the **--compiler** parameter.
   aeproject compile --compiler http://localhost:3080
   ```
 
-The compiler allows you to include other **.aes** files to your contract by using the reserved word **include** and speciying the path to the file. You can add contracts, as well as namespaces.
+The compiler allows you to include other **.aes** files to your contract by using the reserved word **include** and specifying the path to the file. You can add contracts, as well as namespaces.
+PS: Also compiler support some default sophia's libraries. More info how to use these libraries 'List.aes', 'Option.aes', 'Func.aes', 'Pair.aes' and 'Triple.aes' can find here: https://github.com/aeternity/protocol/blob/master/contracts/sophia_stdlib.md
```
Comment (HristiyanG) [inline doc/developer-documentation/aeproject-cli/compile.md]: can find here -> can be found here

File: packages/aeproject-utils/utils/aeproject-utils.js [hunk 2]
```diff
@@ -239,18 +240,39 @@ function getDependencies (contractContent, contractPath) {
...
         dependencyContractPath = path.resolve(`${ contractPath[0] }/${ dependencyFromContract[1] }`)
-        dependencyContractContent = fs.readFileSync(dependencyContractPath, 'utf-8')
-        actualContract = getActualContract(dependencyContractContent)
-        dependencies[dependencyFromContract[1]] = actualContract;
 
-        Object.assign(dependencies, getDependencies(dependencyContractContent, dependencyContractPath))
+        try {
+            dependencyContractContent = fs.readFileSync(dependencyContractPath, 'utf-8');
+        } catch (error) {
+            console.log(`File to include '${ dependencyFromContract[1] }' not found. Check your path or it is from sophia default library`);
```
Comment (HristiyanG) [inline packages/aeproject-utils/utils/aeproject-utils.js]: sophia -> Sophia

File: test/commands-tests/compile.js [hunk 3]
```diff
@@ -52,9 +53,20 @@ describe('AEproject Compile', () => {
             assert.include(result, expectedCompileResultExampleContract);
         })
 
-        it('Should NOT compile contracts with --compiler argument - invalid one ', async () => {
```
Comment (HristiyanG) [inline test/commands-tests/compile.js]: Isn't this test necessary  anymore?

File: doc/developer-documentation/getting-started.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ In a project folder:
 aeproject node
 ```
 
-To spwan a temporary local chain, use `aeproject node --stop` to stop it.
+To spawn a temporary local chain, use `aeproject node --stop` to stop it.
```
Comment (maptuhec) [inline doc/developer-documentation/getting-started.md]: This sentence doesn't make any sense. Should it something like this "To stop already spawned local node use ..." ?

**Pull request merged**